packages/orbit-components/src/CarrierLogo/README.md

```diff
@@ -32,7 +33,7 @@ Table below contains all types of the props available for object in Carrier arra
 | Name     | Type            | Description                                                                                   |
 | :------- | :-------------- | :-------------------------------------------------------------------------------------------- |
 | **code** | `string`        | The IATA code of the Carrier, defines which logo will be rendered.                            |
-| name     | `string`        | The name of the Carrier, mainly for information.                                              |
+| **name** | `string`        | The name of the Carrier, used for alt text.                                                   |
```

Comment:
It is also used as title, shall we mention it? 

Should we even switch alt with ariaLabel? Although this is out of the scope. Feel free to disregard.

Reply:
It was not clear to me, so I didn't touch it much. I found [this article](https://phuoc.ng/collection/this-vs-that/alt-vs-aria-label/) interesting. For our current needs, I'd say alt is fine.

I'll add info about `title`

Reply:
Yes, I agree about alt, it's required for img and needed.

I wrote it wrong in the beginning, I meant title vs aria-label.

Now that I'm thinking, perhaps we don't even need the title at all? Although it provides visual information via tooltip, which in this case might be useful to know the carrier name 🤔

Reply:
Accessibility wise, we don't need title. It doesn't harm to keep it, imo